virttest/qemu_vm.py

```diff
@@ -4396,7 +4396,7 @@ def get_vhost_threads(self, vhost_thread_pattern):
             int(_)
             for _ in re.findall(
                 vhost_thread_pattern % self.get_pid(),
-                process.run("ps aux", verbose=False).stdout_text,
+                process.run("ps -eL", verbose=False).stdout_text,
```

Comment:
The help info from ps:
```
# ps --help threads

Usage:
 ps [options]

Show threads:
  H                   as if they were processes
 -L                   possibly with LWP and NLWP columns
 -m, m                after processes
 -T                   possibly with SPID column
```

I think `ps -e` is enough in this place. the `-L` option has different behavior in different OS. With this change. We should also test multiqueues cases.

Reply:
I just confirmed with the developer that we need to get the LWP values for the vhost threads. Additionally, I checked on the latest system and found that there are no vhost thread ids returned by the “ps -e” command. Therefore, “ps -eL” is needed in the code and I also update the pattern. please help review. 

on rhel9.5:
# ps -eL |grep vhost
   2922    2922 ?        00:00:00 vhost-2918
   2923    2923 ?        00:00:00 vhost-2918
   2924    2924 ?        00:00:00 vhost-2918
   2925    2925 ?        00:00:00 vhost-2918
# ps -e |grep vhost
   2922 ?        00:00:00 vhost-2918
   2923 ?        00:00:00 vhost-2918
   2924 ?        00:00:00 vhost-2918
   2925 ?        00:00:00 vhost-2918
   
 on latest system:
# ps -e |grep vhost
# ps -eL |grep vhost
 186773  186784 pts/1    00:00:00 vhost-186773
 186773  186785 pts/1    00:00:00 vhost-186773
 186773  186786 pts/1    00:00:00 vhost-186773
 186773  186788 pts/1    00:00:00 vhost-186773

Reply:
Ok, this means the kernel behavior changes from 1 vhost per process to 1 vhost per thread. However, I see that you modified the pattern to match LWP instead of PID, which is different from the original pattern, can you explain?

Reply:
From ps man page: LWP  means light weight process (thread) ID of the dispatchable entity (alias spid, tid).

As I mentioned before, I confirmed with developer, we need to check the vhost thread ID from LWP. and also you can see the output on latest system with a vm with 4 queues, the pid is the only one but four LWP are the different,  so, the pattern should be adjusted to correctly match the LWP.

Reply:
Some means you want TID, not PID now. This should be a change, we need to update the doc string.

Reply:
I have updated the both doc string and git commit. please help review.